src/stylesheets/components/_article.scss

```diff
@@ -44,4 +44,41 @@ article {
       }
     }
   }
+
+  & + aside .block .list--pipe:last-child > li:last-child {
```

Comment:
This would be useful not to be specific to article but aside as general. Can make more simple as shown below and moved into lists.scss

`aside .list--pipe li:last-child {
  border-bottom: 0;
  padding-bottom: 0;
}`